handlers/main.yml

```diff
@@ -1,14 +1,38 @@
 ---
 
 - name: reload systemd
-  systemd: daemon_reload=yes
+  systemd:
+    daemon_reload: yes
   when: ansible_distribution_release in ['bionic'] and ansible_service_mgr == 'systemd'
 
 - name: reload supervisor
   command: supervisorctl reload
 
 - name: restart nginx
-  service: name=nginx state=restarted
+  service:
+    name: nginx
+    state: restarted
+
+- name: start mysql
+  service:
+    name: mysql
+    state: started
+
+- name: start postgresql
+  service:
+    name: postgresql
+    state: started
 
 - name: start redis
-  service: name=redis state=started
+  service:
+    name: redis
+    state: started
+
+- name: update-ca-certificates
+  shell: /usr/sbin/update-ca-certificates
```

Comment:
try to use the command module, which is more limited than shell and hence less prone to security issues: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/command_module.html
If `command` works, we should use it instead of shell.

Reply:
Moved to command for now, but [unable to test it right now](https://github.com/openwisp/ansible-openwisp2/pull/223#discussion_r550759166). If it doesn't work after that problem is fixed, I'll move it back to `shell`.